lib/td/client/model.rb

```diff
@@ -404,16 +405,17 @@ class Job < Model
   # @param [String] db_name
   # @param [Fixnum] duration
   # @param [Fixnum] num_records
-  def initialize(client, job_id, type, query, status=nil, url=nil, debug=nil, start_at=nil, end_at=nil, cpu_time=nil,
-                 result_size=nil, result=nil, result_url=nil, hive_result_schema=nil, priority=nil, retry_limit=nil,
-                 org_name=nil, db_name=nil, duration=nil, num_records=nil)
+  def initialize(client, job_id, type, query, status=nil, url=nil, debug=nil, created_at=nil, start_at=nil,
+                 end_at=nil, cpu_time=nil, result_size=nil, result=nil, result_url=nil, hive_result_schema=nil,
+                 priority=nil, retry_limit=nil, org_name=nil, db_name=nil, duration=nil, num_records=nil)
```

Comment:
When adding `created_at` to this initializer, characters per line increased. So that shifted arguments after `created_at`.